internal/compiler/transforms/compile/css-handler/prefix/index.ts

```diff
@@ -0,0 +1,31 @@
+import {
+	PrefixVisitor,
+	wrapPrefixVisitor,
+} from "@internal/compiler/transforms/compile/css-handler/prefix/utils";
+import {ProjectConfig} from "@internal/project";
+import {UnknownObject} from "@internal/typescript-helpers";
+import {CompilerOptions} from "@internal/compiler";
+import {getBrowser} from "@internal/browser-features";
+
+/* GENERATED:START(hash:cd506990d1f961e2a79cd38e90f7adcdabbb4055,id:main) Everything below is automatically generated. DO NOT MODIFY. Run `./rome run scripts/generated-files/lint-rules` to update. */
+import display from "./prefixes/display";
+import transform from "./prefixes/transform";
+import transition from "./prefixes/transition";
+
+const prefixVisitors: PrefixVisitor<UnknownObject>[] = [
+	...display,
+	...transform,
+	...transition,
+];
+/* GENERATED:END(id:main) */
+
+export default (projectConfig: ProjectConfig, options: CompilerOptions) =>
+	prefixVisitors.map((prefixVisitor) =>
+		wrapPrefixVisitor(
+			prefixVisitor,
+			projectConfig.targets.get(options.target ?? "default")?.map((browserProps) =>
+				getBrowser(browserProps)
+			) ?? [],
+		)
+	)
+;
```

Comment:
The expressions here are pretty complex, and it's hard to tell at a glance how all the `?` interact. Might be clearer like this:

```suggestion
export default (projectConfig: ProjectConfig, options: CompilerOptions) => {
	const propsTargets = projectConfig.targets.get(options.target ?? "default") ?? [];
	const targets = propsTargets.map(props => getBrowser(props));
	return prefixVisitors.map((prefixVisitor) => wrapPrefixVisitor(prefixVisitor, targets));
};
```